APMrover2/mode.cpp

```diff
@@ -351,3 +359,28 @@ void Mode::calc_steering_to_heading(float desired_heading_cd, bool reversed)
     const float steering_out = attitude_control.get_steering_out_angle_error(yaw_error, g2.motors.have_skid_steering(), g2.motors.limit.steer_left, g2.motors.limit.steer_right, reversed);
     g2.motors.set_steering(steering_out * 4500.0f);
 }
+
+// calculate vehicle stopping point using current location, velocity and maximum acceleration
+void Mode::calc_stopping_location(Location& stopping_loc)
+{
+    // default stopping location
+    stopping_loc = rover.current_loc;
+
+    // get current velocity vector and speed
+    Vector2f velocity = ahrs.groundspeed_vector();
+    float speed = velocity.length();
+
+    // avoid divide by zero
+    if (!is_positive(speed)) {
```

Comment:
seems unused ?

Reply:
@khancyr, thanks for the review!  That's a real bug you found, awesome.